ui/desktop/electron-app/config/forge.config.js

```diff
@@ -4,6 +4,9 @@ const { isMac } = require('../src/helpers/platform.js');
 const fs = require('fs');
 const path = require('path');
 
+// When BYPASS_CLI_BUILD we do not copy the CLI to the output binary.app/
+const extraResource = process.env.BYPASS_CLI_BUILD === 'true' ? [] : ['./cli'];
```

Comment:
Do we need another flag here? Could we have just always made the `cli` folder and leave it empty if we don't package the CLI? Or even better can't we actually just check if the directory exists here and conditionally decide whether to include the `extraResource`instead?

Reply:
My initial idea with this approach was to provide an specific flag to guarantee a build with no CLI regardless where the build process was run, but mostly thinking in the pipeline.

But after reading your comment, you are right. This could be simple just checking if the cli directory is empty or not.

Since we already have `SETUP_CLI`, we can control not downloading a CLI, therefore cli directory will be empty.

Thanks for the suggestion, I will implement it without `BYPASS_CLI_BUILD`.